### PR TITLE
add ParticipantsController specs

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: app/models/participant.rb [hunk 1]
```diff
@@ -18,7 +18,10 @@ class Participant < ActiveRecord::Base
   has_many :logins, dependent: :destroy
   has_many :session_events, dependent: :destroy
 
-  STATUS = ["pending", "active", "ineligible"]
+  PENDING = "pending".freeze
+  ACTIVE = "active".freeze
+  INELIGIBLE = "ineligible".freeze
+  STATUS = [PENDING, ACTIVE, INELIGIBLE].freeze
```
Comment (michaelwehrley) [inline app/models/participant.rb]: no one is unfreezing this. :snowflake:

File: spec/controllers/participants_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  let(:valid_participant_params) do
+    { first_name: "f", last_name: "l", study_identifier: "s",
+      family_health_unit_name: "u", family_record_number: "r",
+      phone: "555-555-5555", enrollment_date: Date.today, gender: "male" }
+  end
+
+  let(:invalid_participant_params) do
+    { first_name: nil, last_name: nil, study_identifier: nil,
+      family_health_unit_name: nil, family_record_number: nil,
+      phone: nil, enrollment_date: nil }
```
Comment (michaelwehrley) [inline spec/controllers/participants_controller_spec.rb]: is an invalid phone number something such as "555-555-555-5555"?

Comment (ericcf) [inline spec/controllers/participants_controller_spec.rb]: Currently, the only validation on phone numbers is presence. It needs to be augmented to make it more useful.

In Munoz, I'm using the phonelib gem to confirm that the number **could** be real, and then using Twilio to check on details such as the type of number (mobile, landline, voip) and the carrier.

**Comment by michaelwehrley** [comment]
:eyeglasses: good! :boat:

**Pull request merged**